src/main/scala/me/rexim/morganey/reduction/Computation.scala

```diff
@@ -29,4 +29,7 @@ object Computation {
     override def cancel(): Unit = ()
     override def future: Future[T] = f
   }
+
+  def failed[T](message: String, exception: String => Throwable): Computation[T] =
+    fromFuture(Future.failed(exception(message)))
 }
```

Comment:
Well I think I understand the intention of this. You wanted to have a shortcut for creating failed computation but still be able to customize the message and the exception separately, right? But the result feels a bit weird for me to be honest. :) It feels like taking just an exception is enough, because exception already contains the message. Even `Future.failed` takes only exception.